src/components/table.js

```diff
@@ -181,7 +181,7 @@ export const ariaSort = (sort, field) => {
     return 'none'
   }
   // Otherwise this column is not sortable
-  return null
+  return 'none'
```

Comment:
This function is interesting... essentially it returns something compatible with the aria-sort field, and null is not according to typescript's definition of that field. A little hesitant to change something so widely used, but it seems to function alright, and null wasn't even valid value before.

Reply:
how about returning undefined instead?